src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,28 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.DriverService;
+
+@WebServlet(urlPatterns = "/logout")
+public class LogoutController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance("taxi");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = (Long) req.getSession().getAttribute("driver_id");
+        Driver driver = driverService.get(driverId);
+        req.setAttribute("login", driver.getLogin());
```

Comment:
why do you need this parameter? I guess you just need invalidate session and redirect on login page

Reply:
i was trying to display user's login in header, but did in a wrong way